src/containers/Layout/styles.sass

```diff
@@ -0,0 +1,10 @@
+.map-container
```

Comment:
some find curly braces more helpful when it comes to visual parsing, I'm fine with both approaches. The other's don't stay with SASS because of some "far from the standard" features: http://sass-lang.com/documentation/file.INDENTED_SYNTAX.html#deprecated_syntax